packages/odyssey/src/scss/base/_typography-text.scss

```diff
@@ -113,6 +119,24 @@ small {
   font-size: $size-body-caption;
 }
 
+details {
+  font-size: ms(0);
+
+  p {
+    font-size: ms(0);
+  }
+}
+
+summary {
+  font-size: ms(1);
+  font-weight: 600;
+  cursor: default;
```

Comment:
Normally, this display as `text`, but `summary` is interactive, so I've swapped it to match `button`.